P2CODE/AA1/Telematics/examples/example.jsonld

```diff
@@ -0,0 +1,20 @@
+{
+    "id": "Telematics:a567556a88",
+    "type": "Telematics",
+    "UUID": "2536546894662314",
+    "IMEI": "356307047387773",
+    "messageReceivedTime": "2024-02-07 12:55:13.000",
+    "direction": 220,
+    "engineStatus": 1,
+    "phi": "37.8745952",
+    "lamda": "23.921456",
+    "z": "87",
+    "velocity": "50",
+    "outerVoltage": "27.89",
+    "gpsSignalStrength": "77.3",
+    "satellites": 7,
+    "receivedHash": "547g217h89632478",
+    "@context": [
+      "https://raw.githubusercontent.com/smart-data-models/dataModel.AA1/master/context.jsonld"
```

Comment:
This context will change in the final publication, but do not worry about it. We'll do